explain-views.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Views
+summary: Learn about the execution plan information returned by the `EXPLAIN` statement in TiDB.
+---
+
+# Views
+
```

Comment:
```suggestion

This document introduces the execution plan information returned by the `EXPLAIN` statement in TiDB. Several examples are provided.

```

Reply:
I think this is implied by context. I would prefer if we don't include this to keep the content succinct.